data-manipulation/hashing/fnv/hash-data-using-fnv.yml

```diff
@@ -1,25 +1,33 @@
 rule:
   meta:
-    name: hash data using fnv-1
+    name: hash data using fnv
     namespace: data-manipulation/hashing/fnv
-    author: "@_re_fox"
+    author: 
+      - moritz.raabe@fireeye.com
+      - "@_re_fox"
+    description: can be any Fowler-Noll-Vo (FNV) hash variant, including FNV-1, FNV-1a, FNV-0
     scope: function
     references:
       - https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function
+      - http://isthe.com/chongo/tech/comp/fnv/
+      - https://create.stephan-brumme.com/fnv-hash/
     examples:
       - ad4229879180e267f431ac6666b6a0a2:0x14007B4D4
   features:
     - and:
       - characteristic: loop
```

Comment:
I remember a sample where the loop was unrolled, so I left this out. Thoughts on making it optional? The constants are pretty unique features anyway.

Reply:
That's great feedback, I hadn't seen that in a sample.  Changes made.